docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit/rotate-certificates.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Rotate certificates on YugabyteDB Anywhere
+headerTitle: Rotate certificates
+linkTitle: Rotate certificates
+description: Rotate certificates on YugabyteDB Anywhere.
+headcontent: Rotate certificates used by a universe
+menu:
+  preview_yugabyte-platform:
+    parent: enable-encryption-in-transit
+    identifier: rotate-certificates
+    weight: 30
+type: docs
+---
+
+You can rotate certificates for universes configured with the same type of certificates. This involves replacing existing certificates with new database node certificates.
+
+Before rotating certificates, ensure that you have added the certificates to YugabyteDB Anywhere. Refer to [Add certificates](../add-certificate-self/).
+
+## Rotating certificates
+
+Rotating certificates may require a restart of the YB-Master and YB-TServer processes and in some circumstances can result in downtime.
```

Comment:
```suggestion
```

Given we are clarifying the restart and downtime requirement separately, we can skip this line here